### ISSUE TITLE
Are Resque plugins compatible with Rails 4.2 Active Job?

### ISSUE THREAD

**Issue by popup-user** [resque/resque#1354]
I'm interested in using Resque plugins with Rails 4.2 Active Job, but can't find information on whether it is possible/works or not. I posted about it here:
https://github.com/lantins/resque-lock-timeout/issues/30

Can someone answer on whether and how Resque plugins are compatible with code for Active Job? Do I need to revert back to non-active-job-based Resque to use plugins?

Thanks!

**Comment by steveklabnik** [comment]
I do not know, but this would be a good thing to figure out and document.

**Comment by popup-user** [comment]
In a single test I did on the resque-lock-timeout gem, integration didn't appear to work. I found another project that claims to have done an integration with ActiveJob (though it doesn't seem to be actively maintained), which might serve as one example of how to approach integration: https://github.com/idolweb/activejob-lock . I think I'm going to have to go back from ActiveJob to "plain old Resque" to get plugins that I need - which is unfortunate. Thanks.
